### PR TITLE
Use a better name

### PR DESCRIPTION
**Pull request by idanran**

### CODE REVIEW

File: toc.json [hunk 1]
```diff
@@ -94,8 +94,8 @@
   "node": {
     "name": "npm Modules",
     "children": {
-      "npm_specifiers": "Using Node.js Packages with npm Specifiers",
-      "cdns": "Using Node.js Packages Via CDNs",
+      "npm_specifiers": "Using NPM Packages with npm Specifiers",
+      "cdns": "Using NPM Packages Via CDNs",
```
Comment (dsherret) [inline toc.json:98]:
```suggestion
      "cdns": "Using npm Packages Via CDNs",
```

File: toc.json [hunk 2]
```diff
@@ -94,8 +94,8 @@
   "node": {
     "name": "npm Modules",
     "children": {
-      "npm_specifiers": "Using Node.js Packages with npm Specifiers",
-      "cdns": "Using Node.js Packages Via CDNs",
+      "npm_specifiers": "Using NPM Packages with npm Specifiers",
```
Comment (dsherret) [inline toc.json:97]:
```suggestion
      "npm_specifiers": "Using npm Packages with npm Specifiers",
```

**Comment by dsherret** [comment]
@idanran looks good to me, but the CLA needs to be signed for this to be merged

**Comment by idanran** [comment]
> @idanran looks good to me, but the CLA needs to be signed for this to be merged

Signed now

**Review by kt3k — APPROVED** [review]
LGTM

**Comment by kt3k** [comment]
@idanran Thanks!

**Comment by idanran** [comment]
> @idanran Thanks!

My pleasure!

**Pull request merged**